### DISCUSSION TITLE
ORBOS v3.1 breaking changes

### DISCUSSION THREAD

**Discussion by privateVoit** [caos/orbos#678] [Q&A] [answered]
Hi there!

We've seen that 3.1 comes with some breaking changes. Would it be possible if you guys could update the boom example yaml and list those breaking changes in the release?

Thank you!

**Comment by eliobischof** [comment, marked as answer]
Thanks for reporting this.

We've just released version [v4.0.0](https://github.com/caos/orbos/releases/tag/v4.0.0) with the updated examples, docs and release notes.
